<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>主控服务端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="masterport_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>测试端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="testport_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>热像仪2 IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ip_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>热像仪2 PORT：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
